spec/text-editor-registry-spec.js

```diff
@@ -69,9 +69,9 @@ describe('TextEditorRegistry', function() {
   });
 
   describe('.build', function() {
-    it('constructs a TextEditor with the right parameters based on its path and text', async function() {
-      await atom.packages.activatePackage('language-javascript');
-      await atom.packages.activatePackage('language-c');
+    it('constructs a TextEditor with the right parameters based on its path and text', function() {
+      // await atom.packages.activatePackage('language-javascript');
+      // await atom.packages.activatePackage('language-c');
```

Comment:
What is the goal of this? This does not seem right to me

Reply:
Recently atom has had numerous CI pipeline fails. This is an attempt to review the failing tests and understand why the tests are failing due to timeouts.

This test case doesn't depend on the packages being activated to perform the assertion correctly.

Reply:
To fix the CI failures, you should also review and merge #21081 and #21109. Using these two, we are having a much better CI in our fork. Atom-ide-community CI runs in half the time. 

https://github.com/atom-ide-community/atom

Reply:
It looks like those package activations were in there when the test was first added: https://github.com/atom/atom/commit/09d232ba4108c13d350af49abcd325ae113d7289

At least the JavaScript package looks necessary so it will recognize the language and load the grammar, perhaps the C one is there to test it isn't loading the wrong language?

Reply:
@Arcanemagus unfortunately activating the packages alone don't load the grammar. `setLanguageMode` has to be explicitly called for the grammar to be loaded. If you `log` grammar
https://github.com/atom/atom/blob/613f76d5945d2d58435aa6ec839930c4c5b1f021/src/text-editor-registry.js#L96
it will be `null` when running the test.

Reply:
So shouldn't we do that instead of removing this?